secrets/Cargo.toml

```diff
@@ -0,0 +1,24 @@
+[package]
+name = "libcrux-secrets"
+version.workspace = true
+authors.workspace = true
+license.workspace = true
+homepage.workspace = true
+edition.workspace = true
+repository.workspace = true
+readme.workspace = true
+description = "Libcrux utilities for working with secrets"
+exclude = ["/proofs"]
+
+[dependencies]
+hax-lib.workspace = true
+
+[features]
+check-secret-independence = []
+default = []
+
+[lints.rust]
+unexpected_cfgs = { level = "warn", check-cfg = ['cfg(hax)', 'cfg(eurydice)'] }
```

Comment:
Do we actually use these configs?

Reply:
We do use `cfg(hax)` in our code, but maybe that isn't your question?